### ISSUE TITLE
Head Plugin Not Registered when invoked via elasticsearch-transport-thrift

### ISSUE THREAD

**Issue by gdrocell** [mobz/elasticsearch-head#180]
Hi, 
I've created an issue at the following url too:
https://github.com/elasticsearch/elasticsearch-transport-thrift/issues/41

It explains why I created this issue.  I think it can be fixed if I could manually register the head plugin as a handler in the elasticsearch-transport-thrift code.  The only problem is that the head plugin is written in pure javascript, and elasticsearch-transport-thrift is written in Java.  Do you happen to know how I could go about registering the head plugin manually? or do you think there is a better solution to this problem?

**Comment by mobz** [comment]
I'm not totally sure that this is something you should do. Im' not familiar with thrift, but it looks like a service transport framework, where as head is an application (service consumer). Perhaps [running head locally](https://github.com/mobz/elasticsearch-head#running-with-built-in-server) would be a better solution

**Comment by gdrocell** [comment]
Right, so basically I'm invoking a proxy that I created, and it's running on port 8124.  The proxy uses the thrift transport plugin to invoke elastic search, which also runs over SSL.  The main reason that I need to use the thrift transport is because I already have it running through the SSL.  I can't have the head plugin making any ajax calls or anything that are not going through the secure channel.  I'm also currently trying to avoid using any other plugins to have elasticsearch run over https, but that could be a solution too.  

Do you know if I run the head plugin locally and have it run over https?

**Comment by gdrocell** [comment]
When I run the head plugin locally, the head plugin gets a cluster health: not connected.  This happens when I invoke directly via http://localhost:9100.
